src/components/SimpleLink.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <a :href="`#${target}`" class="simple-link">
+  <a
+      :href
```

Comment:
This is a prop-binding shorthand that works if prop and html attribute share the same exact string.